### PR TITLE
[test] Remove retired systems Kesch and Tiger

### PR DESCRIPTION
**Pull request by kraushm**
`kesch[:cn,pn]`  was removed from all tests and configuration files, since the system was taken out of production.

### CODE REVIEW

File: cscs-checks/libraries/math/scalapack_compile_run.py [hunk 1]
```diff
@@ -15,19 +15,12 @@ def __init__(self, linkage):
         self.sourcesdir = os.path.join(self.current_system.resourcesdir,
                                        'scalapack')
         self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:mc',
-                              'dom:gpu', 'kesch:cn']
+                              'dom:gpu']
```
Comment (vkarak) [inline cscs-checks/libraries/math/scalapack_compile_run.py:18]: Can't this fit in the line above?

File: cscs-checks/mch/automatic_arrays_acc.py [hunk 2]
```diff
@@ -11,20 +11,11 @@
 @rfm.simple_test
 class AutomaticArraysCheck(rfm.RegressionTest):
     def __init__(self):
-        self.valid_systems = ['daint:gpu', 'dom:gpu', 'kesch:cn', 'tiger:gpu',
+        self.valid_systems = ['daint:gpu', 'dom:gpu', 'tiger:gpu',
```
Comment (vkarak) [inline cscs-checks/mch/automatic_arrays_acc.py:14]: Remove also tiger, it's also retired.

File: cscs-checks/mch/collectives_halo.py [hunk 3]
```diff
@@ -9,8 +9,9 @@
 
 class CollectivesBaseTest(rfm.RegressionTest):
     def __init__(self, variant, bench_reference):
-        self.valid_systems = ['dom:gpu', 'daint:gpu', 'kesch:cn', 'tiger:gpu',
-                              'arolla:cn', 'tsa:cn']
+        self.valid_systems = ['dom:gpu', 'daint:gpu', 'tiger:gpu', 'arolla:cn',
```
Comment (vkarak) [inline cscs-checks/mch/collectives_halo.py:12]: Remove tiger as well.

File: cscs-checks/mch/collectives_halo.py [hunk 4]
```diff
@@ -9,8 +9,9 @@
...
+                              'tsa:cn']
+
```
Comment (vkarak) [inline cscs-checks/mch/collectives_halo.py:14]: Remove this line.

File: cscs-checks/mch/cuda_stress_test.py [hunk 5]
```diff
@@ -11,13 +11,9 @@
 class CudaStressTest(rfm.RegressionTest):
     def __init__(self):
         self.descr = 'MCH CUDA stress test'
-        self.valid_systems = ['daint:gpu', 'dom:gpu', 'kesch:cn', 'tiger:gpu',
+        self.valid_systems = ['daint:gpu', 'dom:gpu', 'tiger:gpu',
```
Comment (vkarak) [inline cscs-checks/mch/cuda_stress_test.py:14]: Remove tiger as well.

**Review by vkarak — CHANGES_REQUESTED** [review]
Lgtm overall. I have very minor comments and can you please remove "tiger" from everywhere as well?

File: cscs-checks/prgenv/openacc_checks.py [hunk 6]
```diff
@@ -18,28 +17,18 @@ def __init__(self, variant):
...
+        self.valid_systems = ['daint:gpu', 'dom:gpu', 'arolla:cn', 'tsa:cn']
         self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-pgi']
         self.sourcesdir = 'src/openacc'
         if self.num_tasks == 1:
             self.sourcepath = 'vecAdd_openacc_nompi.f90'
-            if self.current_system.name == 'kesch':
-                self.valid_prog_environs = ['PrgEnv-cray-nompi',
-                                            'PrgEnv-pgi-nompi']
-            elif self.current_system.name == 'tsa':
+            if self.current_system.name == 'tsa':
```
Comment (vkarak) [inline cscs-checks/prgenv/openacc_checks.py:25]: Why is this only for Tsa?

**Comment by vkarak** [comment]
@kraushm Can you also check why we have two failing tests?

File: cscs-checks/prgenv/openacc_checks.py (hunk 6 above)
Comment (kraushm) [reply cscs-checks/prgenv/openacc_checks.py]: Indeed, this was incomplete.

**Review by vkarak — APPROVED** [review]
lgtm

**Comment by vkarak** [comment]
@jenkins-cscs retry daint

**Pull request merged**